src/HttpHeader.cc

```diff
@@ -400,9 +400,11 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
         const char *field_start = field_ptr;
         const char *field_end;
 
+        int seenLines = 0;
```

Comment:
Nit: Let's use a more fitting and safer size_t for counting things in a continues memory block:

```suggestion
        size_t lines = 0;
```

Reply:
Done in 1a826d532